projects/vdk-plugins/vdk-meta-jobs/tests/test_meta_dag.py

```diff
@@ -67,3 +70,46 @@ def mock_get_finished_job_names(*args, **kwargs):
 
     # check for busyloop (this would have been called hundreds of times if there is busyloop bug)
     assert calls[0] <= 4
+
+
+def test_build_dag_over_max_starting_jobs_sad_case():
+    job1 = dict(job_name="job1", depends_on=[])
```

Comment:
You can init these using a loop or list comprehension.

Reply:
Done.